lib/OrdinaryDiffEqCore/src/doc_utils.jl

```diff
@@ -62,15 +60,48 @@ function explicit_rk_docstring(description::String,
     keyword_default = """
     stage_limiter! = OrdinaryDiffEq.trivial_limiter!,
     step_limiter! = OrdinaryDiffEq.trivial_limiter!,
+    thread = OrdinaryDiffEq.False(),
     """ * extra_keyword_default
 
     keyword_default_description = """
     - `stage_limiter!`: function of the form `limiter!(u, integrator, p, t)`
     - `step_limiter!`: function of the form `limiter!(u, integrator, p, t)`
+    - `thread`: determines whether internal broadcasting on appropriate CPU arrays should be serial (`thread = OrdinaryDiffEq.False()`) or use multiple threads (`thread = OrdinaryDiffEq.True()`) when Julia is started with multiple threads.
     """ * extra_keyword_description
 
     generic_solver_docstring(
         description, name, "Explicit Runge-Kutta Method. ", references,
         keyword_default_description, keyword_default
     )
 end
+function differentiation_rk_docstring(description::String,
+    name::String,
+    solver_class::String;
+    references::String = "",
+    extra_keyword_description::String = "",
+    extra_keyword_default::String = "")
+    keyword_default = """
+    chunk_size = Val{0}(),
+    autodiff = true,
+    standardtag = Val{true}(),
+    concrete_jac = nothing,
+    diff_type = Val{:forward},
+    linsolve = nothing,
+    precs = DEFAULT_PRECS,
```

Comment:
@ChrisRackauckas I'll make these private for each solver (not only here) unless you object.